src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -10766,6 +10766,145 @@ .maxstack 8
             CompileAndVerify(comp, expectedOutput: "System.Action`2[System.Int32[],System.Int32]");
         }
 
+        [ConditionalFact(typeof(CoreClrOnly))]
+        [WorkItem(63565, "https://github.com/dotnet/roslyn/issues/63565")]
+        public void SynthesizedDelegateTypes_UnscopedRefAttribute_01()
+        {
+            string source = """
+                using System;
+                using System.Diagnostics.CodeAnalysis;
+                class Program
+                {
+                    static ref int F1(ref int x, ref int y) => ref x;
+                    static ref int F2(ref int x, [UnscopedRef] ref int y) => ref x;
+                    static ref int F3([UnscopedRef] ref int x, ref int y) => ref x;
+                    static void Main()
+                    {
+                        var d1 = F1;
+                        var d2 = F2;
+                        var d3 = F3;
+                        var d4 = ([UnscopedRef] ref int x, ref int y) => ref x;
```

Comment:
>([UnscopedRef] ref int x, ref int y) => ref x; [](http://example.com/codeflow?start=33&length=46)

It might be good to separate method group scenarios and lambda scenarios, to make sure they are properly handled on their own.